os/configs/drafteros-firecracker-x86_64_pvm_defconfig

```diff
@@ -1,6 +1,7 @@
 BR2_x86_64=y
 BR2_KERNEL_HEADERS_VERSION=y
-BR2_DEFAULT_KERNEL_VERSION="6.8.12"
+BR2_DEFAULT_KERNEL_VERSION="6.7.0-rc6"
```

Comment:
Wait why is this change here?

Reply:
Bug fix! Didn't notice it earlier since CI/CD was down, but the newer version of Buildroot actually checks the kernel header versions, so I've specified the proper kernel release string :)